### ISSUE TITLE
update-notifier: Cannot use namespace 'ConfigStore' as a type

### ISSUE THREAD

**Issue by jstorm31** [AckeeCZ/lokse#77]
Adding `lokse` in node.js project caused type error in `update-notifier` package:

```
node_modules/@types/update-notifier/index.d.ts:21:26 - error TS2709: Cannot use namespace 'ConfigStore' as a type.

21         readonly config: ConfigStore;
                            ~~~~~~~~~~~


Found 1 error in node_modules/@types/update-notifier/index.d.ts:21
```

Installing `@types/update-notifier` version `6.x` helped. I could update `update-notifier` package in `lokse`, but the newest verson `6.x` runs purely on ESM modules, which might be a breaking change and wouldn't be usable in non ESM projects.

Any ideas how to solve this? 🤔 

## Versions

- node.js `16.17.1`
- TypeScript `4.8.3`
- npm: `8.15.0`

**Comment by jstorm31** [comment]
After upgrading the project to node `18.12` we got rid of the issue 🤔 But not sure that was the cause 😄

**Comment by horaklukas** [comment]
what are the steps to reproduce it? Just install the package? Or run some command?

**Comment by grissius** [comment]
I am sorry to report I faild to reproduced the problem with both v18 and v16 on bare install and both installed the dep under lokse

```
lokse-reproduce-18> node -v && npm -v
v18.12.0
8.19.2
lokse-reproduce-18> npm i typescript lokse
...
lokse-reproduce-18> npx tsc --init
...
lokse-reproduce-18> echo 'console.log("sad lokse")' > index.ts
lokse-reproduce-18> npx tsc
lokse-reproduce-18> npm ls @types/update-notifier
lokse-reproduce-18@ /tmp/lokse-reproduce-18
└─┬ lokse@2.2.1
  └── @types/update-notifier@5.1.0
```

```
smolijar@latitude /tmp> cd lokse-reproduce-16
lokse-reproduce-16> node -v && npm -v
v16.17.1
8.15.0
lokse-reproduce-16> npm i typescript lokse
...
lokse-reproduce-16> npx tsc --init
...
lokse-reproduce-16> echo 'console.log("sad lokse")' > index.ts
lokse-reproduce-16> npx tsc
lokse-reproduce-16> npm ls @types/update-notifier
lokse-reproduce-16@ /tmp/lokse-reproduce-16
└─┬ lokse@2.2.1
  └── @types/update-notifier@5.1.0
```

**Comment by horaklukas** [comment]
I reproduced it, although I didn't want to 😅 https://github.com/AckeeCZ/lokse/actions/runs/3387926001/jobs/5629288973#step:6:69

Anyway I decided to switch to [update notifier from oclif team](https://github.com/oclif/plugin-warn-if-update-available/). It provides same functionality, but should be better compatible with the framework. We're not locked to the `update-notifier` package as it is not core functionality, neither it brings some additional value.
